### PR TITLE
fix(components/packages): update package group name to `@skyux/packages`

### PR DESCRIPTION
**Pull request by johnhwhite**
[AB#2963503](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2963503)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/2375/storybook/)

Component Storybooks:

- _(no component storybooks affected in this pr)_ 

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/2375/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/2375/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/2375/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/2375/playground/)

File: libs/components/packages/package.json [hunk 1]
```diff
@@ -88,7 +88,8 @@
       "ag-grid-community": "~31.2.0",
       "ag-grid-enterprise": "~31.2.0",
       "autonumeric": "^4.10.5"
-    }
+    },
+    "packageGroupName": "@skyux/packages"
```
Comment (Blackbaud-SteveBrush) [inline libs/components/packages/package.json]: From [their docs](https://github.com/angular/angular-cli/blob/438b53075091deddbf5a05a4dc9e90c75afcabcd/docs/specifications/update.md?plain=1#L75), it looks like we need to include `@skyux/packages` as the top-most item in the `packageGroup` array, then we don't need to set this property?
> The packageGroupName needs to be part of the packageGroup and should be a valid package name.

Comment (Blackbaud-SteveBrush) [reply libs/components/packages/package.json]: It appears that Angular CLI also does this: https://github.com/angular/angular-cli/blob/main/packages/angular/cli/package.json#L46

File: libs/components/packages/project.json [hunk 2]
```diff
@@ -27,13 +27,14 @@
...
-      "options": {
-        "command": "ts-node --project ./scripts/tsconfig.json ./scripts/postbuild-packages.ts"
+        "assets": [
+          "libs/components/packages/collection.json",
+          "libs/components/packages/src/polyfills.js",
+          "libs/components/packages/src/schematics/migrations/migration-collection.json",
+          "libs/components/packages/src/schematics/**/schema.json",
+          "libs/components/packages/src/schematics/**/*.template",
+          "libs/components/packages/*.md"
+        ]
```
Comment (Blackbaud-SteveBrush) [inline libs/components/packages/project.json:37]: Beautiful.

File: libs/components/packages/package.json (hunk 1 above)
Comment (Blackbaud-SteveBrush) [reply libs/components/packages/package.json]: Can you make this same change to `@skyux-sdk/eslint-config` and `@skyux-sdk/prettier-schematics` update groups?

Comment (johnhwhite) [reply libs/components/packages/package.json]: Good call. Updated.

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**